pages/guide.py

```diff
@@ -271,12 +271,16 @@ def _get_field_val(self, field: str, field_type: str) -> Any:
     elif field_type == 'str':
       return self.form.get(field)
     elif field_type == 'split_str':
-      val = self.split_input(field, delim=',')
       if field == 'rollout_platforms':
         val = self.form.getlist(field)
+        # Occasionally, input will give an empty string as the first element.
+        # It needs to be removed.
+        if val and len(val[0]) == 0:
```

Comment:
This would be more clear as `val and val[0] == ''`.

Reply:
Good idea - adjusted 🙂